packages/angular_devkit/benchmark/src/main_spec.ts

```diff
@@ -142,4 +143,13 @@ describe('benchmark binary', () => {
     stdout.lines.forEach(line => expect(line).toMatch(/^\[abc\]/));
     expect(res).toEqual(0);
   });
+
+  it('uses watch-script and watch-matcher', async () => {
+    const benchmarkScript = require.resolve(join(__dirname, './test/watch-test-cmd.js'));
+    const watchScript = require.resolve(join(__dirname, './test/watch-test-script.js'));
+    const args = ['--watch-matcher', 'Complete', '--watch-script', watchScript, '--', 'node', benchmarkScript];
```

Comment:
Shouldn't `watchScript` here also be prefixed with `node`? Or is the script always a node script? 

I think it's fine to limit it to a runnable script instead of a command with args though, mainly because handling multiple commands inside one command is hairy business.

Reply:
Yes it is always a node script. Unless using the API where you can provide a `Command` object.

Reply:
Can you mention on the help usage it's a node script?